docs/source/conf.py

```diff
@@ -306,4 +306,6 @@ def setup(app):
 .. |removed| replace:: :red:`removed`
 .. |fixed| replace:: :purple:`fixed`
 
+.. |unit_url| replace:: http://<esp easy ip/host name>/
```

Comment:
That `/` is confusing in this context :)

Reply:
I see what you mean. I'll just remove it 😉